pom.xml

```diff
@@ -190,7 +190,7 @@
         <commons-pool.version>1.6</commons-pool.version>
         <commons-validator.version>1.6</commons-validator.version>
         <components-font-awesome.version>4.7.0</components-font-awesome.version>
-        <countryconverter.version>0.1.3</countryconverter.version>
```

Comment:
❗️ 	I don't think we should use a snapshot version. `0.1.4` was released an hour ago